File: Clases_Instancias/1/estudiante.py
class Estudiante:
    def __init__(self, nombre, edad, promedio):
        self.nombre = nombre
        self.edad = edad
        self.promedio = promedio

    def mostrarInfo(self):
        print(f"Nombre: {self.nombre}")
        print(f"Edad: {self.edad}")
        print(f"Promedio: {self.promedio}")

# Ejemplo de uso
if __name__ == "__main__":
    estudiante1 = Estudiante("Oscar", 20, 4.5)
    estudiante1.mostrarInfo()

File: Clases_Instancias/1/estudiante.cpp
#include <iostream>
#include <string>

using namespace std;

class Estudiante {
public:
    string nombre;
    int edad;
    float promedio;

    Estudiante(string n, int e, float p) : nombre(n), edad(e), promedio(p) {}

    void mostrarInfo() {
        cout << "Nombre: " << nombre << endl;
        cout << "Edad: " << edad << endl;
        cout << "Promedio: " << promedio << endl;
    }
};

int main() {
    Estudiante estudiante1("Oscar", 20, 4.5);
    estudiante1.mostrarInfo();
    return 0;
}
